vercel.json

```diff
@@ -0,0 +1,755 @@
+    },
+    {
+      "source": "/guides/tina-cloud(.*)/",
+      "destination": "/guides/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/features/extending-tina/",
+      "destination": "/docs/advanced/extending-tina/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/mdx/",
+      "destination": "/docs/editing/markdown",
+      "permanent": true
+    },
+    {
+      "source": "/docs/tinacms-package/",
+      "destination": "/docs/tinacms-context",
+      "permanent": true
+    },
+    {
+      "source": "/docs/cli/",
+      "destination": "/docs/cli-overview",
+      "permanent": true
+    },
+    {
+      "source": "/docs/schema-advanced/",
+      "destination": "/docs/graphql/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/graphql/",
+      "destination": "/docs/graphql/overview",
+      "permanent": true
+    },
+    {
+      "source": "/docs/releases(.*)/",
+      "destination": "/docs/legacy-redirect",
+      "permanent": true
+    },
+    {
+      "source": "/docs/advanced/extending-tina/",
+      "destination": "/docs/extending-tina/overview/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/advanced/configuring-field-plugin/",
+      "destination": "/docs/extending-tina/custom-field-components/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/advanced/extending-field-plugin/",
+      "destination": "/docs/extending-tina/advanced/extending-field-plugin/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/advanced/creating-field-component/",
+      "destination": "/docs/extending-tina/advanced/creating-field-component/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/advanced/customizing-forms/",
+      "destination": "/docs/extending-tina/customizing-forms/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/advanced/edit-state/",
+      "destination": "/docs/extending-tina/edit-state/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/media-cloudinary/",
+      "destination": "/docs/reference/media/external/cloudinary/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/tina-cloud/data-layer/",
+      "destination": "/docs/reference/content-api/data-layer/",
+      "permanent": true
+    },
+    {
+      "source": "/guides/tinacms/gatsby-data-fetching/guide/",
+      "destination": "/guides/tinacms/gatsby-data-fetching/",
+      "permanent": true
+    },
+    {
+      "source": "/guides/tinacms/non-react-based-ssg/guide/",
+      "destination": "/docs/frameworks/other/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/editing/active-field-indicator/",
+      "destination": "/docs/contextual-editing/react/#click-to-edit-experimental",
+      "permanent": true
+    },
+    {
+      "source": "/guides/tinacms/nextjs-internationalization/guide/",
+      "destination": "/guides/tinacms/adding-internationalization-to-tina/guide",
+      "permanent": true
+    },
```

Comment:
guides 404

Reply:
fixed to docs/guides/nextjs-internationalization/